src/apiNetworkProvider.ts

```diff
@@ -16,17 +16,21 @@ import { DefinitionOfFungibleTokenOnNetwork, DefinitionOfTokenCollectionOnNetwor
 import { FungibleTokenOfAccountOnNetwork, NonFungibleTokenOfAccountOnNetwork } from "./tokens";
 import { TransactionOnNetwork, prepareTransactionForBroadcasting } from "./transactions";
 import { TransactionStatus } from "./transactionStatus";
+import { setUserAgent } from "./userAgent";
+import { ExtendedAxiosRequestConfig } from "./NetworkProviderConfig";
 
 // TODO: Find & remove duplicate code between "ProxyNetworkProvider" and "ApiNetworkProvider".
 export class ApiNetworkProvider implements INetworkProvider {
     private url: string;
-    private config: AxiosRequestConfig;
+    private config: ExtendedAxiosRequestConfig;
     private backingProxyNetworkProvider;
+    private userAgentPrefix = 'sdk-network-providers/api'
```

Comment:
In the near future, we will unify the JS libraries. Thus, let's use `multiversx-sdk/api` (or something similar - "multiversx" is necessary, to stand out from other possible libraries that talk to MultiversX APIs).

Constants can also be moved to `constants.ts`.

Reply:
done